canary/producer-c/canary/KvsProducerSampleCloudwatch.cpp

```diff
@@ -330,7 +350,12 @@ INT32 main(INT32 argc, CHAR* argv[])
                 lastKeyFrameTimestamp = frame.presentationTs;
             }
             CHK_STATUS(putKinesisVideoFrame(streamHandle, &frame));
-
+            startUpLatency = (DOUBLE)(GETTIME() - startTime) / (DOUBLE) HUNDREDS_OF_NANOS_IN_A_MILLISECOND;
```

Comment:
Move this into the if loop as it's only used when publishing the metrics. No need to calculate it for no reason.